include/brickengine/collision_detector.hpp

```diff
@@ -40,12 +40,27 @@ Inspector Collision Detector
 #include "brickengine/enum/axis.hpp"
 #include "brickengine/enum/direction.hpp"
 
+struct CollisionReturnValues {
+    CollisionReturnValues(double spaceLeft, int objectId, bool isTrigger) : spaceLeft(spaceLeft), objectId(objectId), isTrigger(isTrigger) {}
```

Comment:
![image](https://user-images.githubusercontent.com/21974974/68787526-02a1fc80-0642-11ea-9664-de5f5af75d03.png)
Any sneks in here?